readme.md

```diff
@@ -53,7 +53,71 @@ Directory names should be lowercase. Filenames should be singular.
 *todo: evaluate common prefix like `pagelib` and BEM naming.*
 
 ## Development setup and workflow
-*Coming soon.*
+The Android and iOS Wikipedia apps make use of the `wikimedia-page-library`. At some point their respective build phases invoke `npm install` which causes a *published* version of this library to be retrieved and used in their respective builds. 
```

Comment:
But even in your example, if you set your editor to wrap lines wouldn't it just be able to reflow the text if it didn't have all these breaks at 80? That's how Xcode is configured out of the box iirc - though I don't use the side-by-side view very often.